test/compiler/parsing/index.js

```diff
@@ -21,6 +21,7 @@ function createCheck(suite, ast) {
   let expected;
   try {
     expected = readFileSync(expectedFile, "utf8");
+    expected = JSON.stringify(JSON.parse(expected), null, 2);
```

Comment:
prettier was formatting my expected JSON files, causing the tests to fail. This change re-formats the expected JSON to match the actual.

Reply:
We can probably ignore them in the prettier configuration? I guess you are using the integration in your editor because I don't have that issue?

Reply:
yes, it is due to prettier - however, I think it makes the tests a bit less fragile if we don't have to formal the expected JSON in a specific way.

Reply:
Ok, yes that's right.